Resources/Prototypes/Catalog/Fills/Crates/food.yml

```diff
@@ -20,7 +21,8 @@
   - type: StorageFill
     contents:
     - id: FoodBoxPizzaFilled
-      amount: 16
+      amount: 15
+    - id: FoodBoxPizzaCotton
```

Comment:
Wouldn't it make sense to have like, two or three of these in the large crate?

Reply:
It is something I considered. Right now we have about 2/16 on average with this change which feels like a fair perportion given its about 1/8 species. If anything, the small pizza box overrepesents cotton pizzas, but I decided to prioritize there is at least something for everyone in this case. I am happy to change it though if you feel 2-3 feels more appropriate.

Reply:
Oh right the thing can already spawn one, right.

Reply:
I can add a comment to clarify the reasoning if needed

Reply:
Nah you're good.